pages/1.11/release-notes/1.11.2/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.2
+title: Release Notes for 1.11.2
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.2
+---
+
+These are the release notes for DC/OS 1.11.2.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.2/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.2 includes the following:
+
+- Apache Mesos 1.5.1-dev [change log](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
+- Marathon 1.6.392 [change log](https://github.com/mesosphere/marathon/releases).
+- Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
+
+
+# Issues Fixed in DC/OS 1.11.2
+
+- DCOS-14199 - Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file.
+- DCOS-20514 - Added licensing information to the diagnostics bundle.
+- DCOS-21596 - Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
+- DCOS_OSS-2360 - DC/OS Metrics: metric names are sanitized for better compatibility with Prometheus.
+- DCOS_OSS-2378 - DC/OS Net: Improved stability of distribution protocol over TLS. [enterprise type="inline" size="small" /]
+- DC/OS UI: Incorporated [multiple](https://github.com/dcos/dcos/pull/2799) fixes and improvements.
+
+
+# Notable Changes in DC/OS 1.11.2
+
+- DCOS-21958 - Admin Router on the DC/OS master nodes now does not support TLS 1.1 and the 3DES bulk encryption algorithm anymore by default.
+- DCOS-22326 - Disabled the insecure 3DES bulk encryption algorithm and TLS 1.1 protocol by default for Master Admin Router. [enterprise type="inline" size="small" /] 
+- DCOS-29122 - Support for launching services in a remote region.
+- DCOS-29634 - Implement region validation and config-sniffing in SDK.
+- DCOS_OSS-2377 - Agent operator API: Displayed resource provider resources in GET_RESOURCE_PROVIDER response.
+- QUALITY-2006 - RHEL 7.4 with Docker EE 17.06.2 is supported.
+- QUALITY-2007 - RHEL 7.4 with Docker 17.12.1-ce is supported. 
+- QUALITY-2057 - CentOS 7.4 with Docker EE 17.06.2 is supported.
+
+
+**Note:** 
+1. Previous 1.11 point releases are not supported for CoreOS 1688.5.3.
+
+2. Implemented region awareness support for SDK based services are implemented in this release. [enterprise type="inline" size="small" /]
+
+3. New Docker versions supported on RHEL 7.4 and CoreOS 1688.5.3. See [compatibility matrix](https://docs.mesosphere.com/version-policy/) for further information.
```

Comment:
To be approved by @gauripowale .

Reply:
Re-tested RHEL 7.4 against 1.11.2, which passed all tests. Please include this in the RN & in the compatibility matrix. You can remove CoreOS 1688.5.3 from the release notes and the compatibility matrix as this fix for this will land in 1.11.3.

Reply:
@gauripowale thank you for the confirmation. I've removed all CoreOS 1688.5.3 entries from 1.11.2 RN documentation.